### PR TITLE
Adm

### PR DESCRIPTION
**Pull request by hiroaki-iwase**
I developed adm tool.
Please check it.

### CODE REVIEW

**Comment by hiroaki-iwase** [comment]
I renamed tool name from adm to roma-adm.
Please check it.

File: bin/roma-adm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+require 'roma/tools/roma-adm'
+
+# opt parse
+opts = OptionParser.new
+opts.banner="usage:#{File.basename($0)} [command] [port No.]"
+opts.on("-h", "--help", "Show this message") {
+  puts opts; exit
+}
+opts.parse!(ARGV)
```
Comment (byplayer) [inline bin/roma-adm]: I think it is better to rescue exception and show usage

Comment (hiroaki-iwase) [inline bin/roma-adm]: Thanks for your check!
That makes sense.
Let me add your advice points.

Comment (hiroaki-iwase) [inline bin/roma-adm]: You think it is better to cover exception not only main logic but also all logic including optpaser and argument check, right?
Now usage has a just few info, so I will improve it.

**Pull request merged**